### DISCUSSION TITLE
Download - Brave browser.

### DISCUSSION THREAD

**Discussion by b4h4w** [regolith-linux/regolith-desktop#1052] [Q&A] [answered]
Hi guys! I'm trying regolith on Debian 12 and noticed I can't seem to have the pop-up windows when downloading / saving files on brave-browser. Is there any specific config file that I need to update to fix this?

NOTE: This is working fine when I switch to xfce4.

**Comment by kgilmer** [comment, marked as answer]
I haven't used that app but this may be related to the xdg-desktop-portal configuration.  Try toggling the install state of `xdg-desktop-portal-regolith` package (if installed, remove. If not installed, install).  Re-login to test to see if the package change effects the problem.

**Reply by b4h4w** [reply]
Yup.  Removing xdg-desktop-portal-regolith does solve this issue. Thank you so much @kgilmer !
